yarn.lock

```diff
@@ -44,6 +48,16 @@
     ipfs-http-client "^51.0.0"
     yup "^1.2.0"
 
+<<<<<<< HEAD
+=======
+"@aragon/osx-commons-subgraph@^0.0.4":
+  version "0.0.4"
+  resolved "https://registry.yarnpkg.com/@aragon/osx-commons-subgraph/-/osx-commons-subgraph-0.0.4.tgz#2aa52f3089d21189c9152d2f3d14c0d7c66d129f"
+  integrity sha512-cqhusJ3HNvMx+t9lXfN+Hy/5ipefNs1Tdxe+y0GvD4qgBMVU4tCbsxOpB9U2JEJNBCzFQj4E/872FFLpIErB4w==
+  dependencies:
+    "@graphprotocol/graph-ts" "0.31.0"
+
+>>>>>>> f7ac8f54 (update commons version)
```

Comment:
We have an unresolved merge conflict here

Reply:
@Rekard0 Another thing that wasn't catched by our github workflows.

Reply:
https://aragonassociation.atlassian.net/browse/OS-1066